pkg/compact/compact_e2e_test.go

```diff
@@ -29,6 +29,10 @@ import (
 	"github.com/thanos-io/thanos/pkg/testutil"
 )
 
+const (
+	BlockDelay = 30 * time.Minute
```

Comment:
Is it necessary to expose this const publicly?

Reply:
Made this change